README.md

```diff
@@ -73,11 +73,11 @@ Watch [use the Angular CLI to transform your Ionic app into a PWA](https://youtu
 
 ## Entity Generator
 
-To generate entities, run `ionic4j entity <name>` or `yo jhipster-ionic:entity <name>`.
+To generate entities, run `ionic4j entity <name>`.
 
 ## Import JDL
 
-To import JDL, run `ionic4j import-jdl <entities.jdl>` or `yo jhipster-ionic:import-jdl <entities.jdl>`.
+To import JDL, run `ionic4j import-jdl <entities.jdl>`.
```

Comment:
Maybe this is a good time to rename `import-jdl` to `jdl` to be consistent with JHipster? 

I'm not sure this is a great idea since JHipster's `jdl` command allows you to create whole apps and this command only allows you to generate entities.